LEGO1/lego/legoomni/src/common/legoanimationmanager.cpp

```diff
@@ -294,10 +294,15 @@ MxResult LegoAnimationManager::LoadScriptInfo(MxS32 p_scriptIndex)
 	return result;
 }
 
-// STUB: LEGO1 0x10060140
-MxBool LegoAnimationManager::FUN_10060140(char* p_name, MxU32& p_index)
+// FUNCTION: LEGO1 0x10060140
+MxBool LegoAnimationManager::IsNotVehicle(char* p_name, MxU32& p_index)
 {
-	// TODO
+	for (MxS32 i = 0; i < sizeof(g_vehicles); i++) {
```

Comment:
`sizeof` is incorrect. It should be `_countof`. The fact that this matches to 100% despite that should be looked into, I'm assuming it has to do with the offset comparison past the structure end. @disinvite could you check why it passes?